apps/signer/main.c

```diff
@@ -204,9 +204,12 @@ main(gint argc, gchar *argv[])
   filesink = gst_element_factory_make("filesink", NULL);
 
   if (!filesrc || !demuxer || !parser || !muxer || !filesink) {
-    g_message(
-        "One or more standard elements in pipeline could not be found - check "
-        "your install.");
+    if (!filesrc) g_message("gStreamer element 'filesrc' not found");
+    if (!demuxer) g_message("gStreamer element '%s' not found", demux_str);
+    if (!parser) g_message("gStreamer element '%sparse' not found", codec_str);
```

Comment:
-> '%s' parser not found

Reply:
But the gStreamer element is called `h264parse` and `h265parse`